README.md

```diff
@@ -82,24 +82,43 @@ See [CONTRIBUTING.md](CONTRIBUTING.md)
 OpenTelemetry Python is under active development.
 
 The library is not yet _generally available_, and releases aren't guaranteed to
-conform to a specific version of the specification.  Future releases will not
-attempt to maintain backwards compatibility with current releases.
+conform to a specific version of the specification. Future releases will not
+attempt to maintain backwards compatibility with previous releases.
 
-The _alpha_ release includes:
+The [v0.1 alpha
+release](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v0.1.0)
+includes:
 
 - Tracing API
 - Tracing SDK
 - Metrics API
 - Metrics SDK
```

Comment:
"Metrics SDK" is present here as well as in alpha v0.2. What is the current status?

Reply:
It's half-implemented. Gauges, exporters, aggregation are all still WIP. I'll change the text.